lib/compat/wordpress-6.5/blocks.php

```diff
@@ -45,3 +45,82 @@ function gutenberg_register_metadata_attribute( $args ) {
 	return $args;
 }
 add_filter( 'register_block_type_args', 'gutenberg_register_metadata_attribute' );
+
+
+if ( ! function_exists( 'gutenberg_process_block_bindings' ) ) {
+	/**
+	 * Process the block bindings attribute.
+	 *
+	 * @param string   $block_content Block Content.
+	 * @param array    $block Block attributes.
+	 * @param WP_Block $block_instance The block instance.
+	 */
+	function gutenberg_process_block_bindings( $block_content, $block, $block_instance ) {
+
+		// Allowed blocks that support block bindings.
+		// TODO: Look for a mechanism to opt-in for this. Maybe adding a property to block attributes?
+		$allowed_blocks = array(
+			'core/paragraph' => array( 'content' ),
+			'core/heading'   => array( 'content' ),
+			'core/image'     => array( 'url', 'title', 'alt' ),
+			'core/button'    => array( 'url', 'text' ),
```

Comment:
In [this pull request](https://github.com/WordPress/gutenberg/pull/58165/files#diff-be8a354ce9c01e85c0fdf04c75d76880cdc6c81992db7a86a7a0515dc11a9833R104), support for the `linkTarget` attribute was added. I believe we have to include it here as well.

Reply:
You're right. Thanks Mario!

Reply:
Fixed in 0fc32626ba84ec6e25eda6fe87661d3c3f95eb74